src/edu/washington/escience/myria/expression/evaluate/PythonUDFEvaluator.java

```diff
@@ -97,22 +100,17 @@ public PythonUDFEvaluator(
    */
   private void initEvaluator() throws DbException {
     ExpressionOperator op = getExpression().getRootExpressionOperator();
-
     String pyFunctionName = ((PyUDFExpression) op).getName();
 
     try {
       if (pyFuncRegistrar != null) {
```

Comment:
Sorry if we went over this already, but when would `pyFuncRegistrar` be null? Also, if the `pyFuncRegistrar` is null, you shouldn't run the rest of the evaluation, right? Right now it seems like it would be doing so...